2020 update: rules and fees

hsnfirooz opened the pull request:
I updated the Programs list, 2020 Tuition fees, application process based on the new rules, and added a few tips in the application section.
Thanks for your awesome guide, Sina!

hsnfirooz committed:
Update README.md

Updating Programs lists

hsnfirooz committed:
Update README.md

The tuition fee and Scholarship section updated.

hsnfirooz committed:
Update README.md

Update the Application section with new rules and a few tips.

ssheikholeslami commented on README.md:
```diff
@@ -41,11 +41,15 @@
+
+* Fintech
+
+* Digital Manufactaring
```
Digital Manufacturing

hsnfirooz replied:
Sorry, my bad.

hsnfirooz replied:
Fixed on 2623995864caea8c50c1602e1265a45273939fa5

ssheikholeslami commented on README.md:
```diff
@@ -107,13 +111,11 @@
 
 در مورد تاریخ اپلای کردن هم دقت کنید که دو بازه برای اپلای کردن مشخص میکنند که اگر میخواهید برای گرفتن اسکالرشیپ درنظرتون بگیرند **حتمن** و **حتمن** باید در بازه‌ی اول اقدام کنید. من سال اولی که برای EIT اپلای کردم یه ربع (پانزده دقیقه!) بعد از تموم شدن بازه‌ی اول درخواستم رو فرستادم، و سر همین قضیه با اینکه پذیرش گرفتم ولی من رو برای گرفتن اسکالرشیپ مدنظر قرار ندادن و سال بعدش دوباره اپلای کردم. ...
-
+یه مزیت دیگه‌ی اقدام کردن در بازه اول اینه که شما میتونین برای دوتا پروگرام اقدام کنید. البته باید برای هر دو تا SOP جداگانه بنویسید ولی مثلا توصیه‌نامه‌هاتون برای هر دوتا برنامه محسوب میشن. البته دقت کنین که الویت انتخاب بر اساس پروگرام اوله و اگه توی اون پذیرفته بشین، درخواست پروگرام دومتون بررسی نمیشه. لذا با الویت و دقیق انتخاب کنین.
```
اولویت

hsnfirooz replied:
این رو واقعا نمیدونستم! فکر میکردم الویت نوشته میشه :‌))

hsnfirooz replied:
Fixed on 2623995864caea8c50c1602e1265a45273939fa5

ssheikholeslami commented on README.md:
```diff
@@ -131,8 +133,8 @@ EIT قصد داره [یک سری کورس و اسپشالیزیشن آنلاین
-اگر اسکالرشیپ گرفتید، با مقداری کار پاره وقت به راحتی میتونید هزینه‌ی تحصیل و زندگی خودتون رو دربیارید. کار دانشجویی خیلی عادی و تعریف‌شده‌ست و اگر مرتبط با رشته‌های خودتون بتونین کار پیدا کنین (که چون دانشجوی دانشگاه‌های معتبر هستید کارِ راحتیه) هم تجربه‌ی خیلی خوبی کسب میکنید هم حقوقش خوبه. TA شدن توی درس‌های دانشگاه‌ها هم حقوق خوبی داره، شاید بد نباشه درس‌هایی که توی دانشگاه‌های مقصدتون ارائه  ...
-
+اگر اسکالرشیپ گرفتید، با مقداری کار پاره وقت به راحتی میتونید هزینه‌ی تحصیل و زندگی خودتون رو دربیارید. کار دانشجویی خیلی عادی و تعریف‌شده‌ست و اگر مرتبط با رشته‌های خودتون بتونین کار پیدا کنین (که چون دانشجوی دانشگاه‌های معتبر هستید کارِ راحتیه) هم تجربه‌ی خیلی خوبی کسب میکنید هم حقوقش خوبه. TA شدن توی درس‌های دانشگاه‌ها هم حقوق خوبی داره، شاید بد نباشه درس‌هایی که توی دانشگاه‌های مقصدتون ارائه  ...
+هزینه ثبت‌نام رو باید قبل از شروع هر ترم پرداخت کنین. این شهریه تغییر میکنه و این لینک مقدار دقیقش مشخصه. زمان ادیت این فایل (سال ۲۰۲۰) شهریه برای هر ترم ۶۰۰۰ یوروه.
```
کدوم لینک؟

hsnfirooz replied:
ظاهرا فراموش کردم لینک رو بذارم. یه "در" هم کم داشته البته.
Fixed on 2623995864caea8c50c1602e1265a45273939fa5

ssheikholeslami reviewed (changes requested):
خیلی مرسی حسین جان

hsnfirooz committed:
Fixed miss spellings and missing link on tuition fee

Fixed all three problems mentioned here:
https://github.com/ssheikholeslami/EIT-Digital-In-Farsi/pull/13#discussion_r417581596
and https://github.com/ssheikholeslami/EIT-Digital-In-Farsi/pull/13#discussion_r417582638
and https://github.com/ssheikholeslami/EIT-Digital-In-Farsi/pull/13#discussion_r417583246

hsnfirooz committed:
update scholarship details

I asked the EIT admission office about the details of renewing the scholarship from merit to excellence and about the tuition wavier.
Check out [this webinar](https://www.gotostage.com/channel/d0a922f37962447d8d9eaa943a1e7db6/recording/f5e76eaaaa2c4191b2098fe7e670f5c0/watch?tos=true&ticket=eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiI0NjM2MDE5NzE2ODA3OTM2NTI0IiwiYXVkIjoiMTAxIiwibHMiOiI2MjUyZjAzMS0xZmMzLTRjMDAtYjJjZi1mMGY4Njg4MzlkNWYiLCJvZ24iOiJnKyIsImV4cCI6MTU4ODA3MTQyMywidHlwZSI6IjEiLCJqdGkiOiIzODQ3Mzc5Mi1hODhkLTQyNmItOTkyMS1jYzlhZGQ2NmQ2NGUiLCJ0ZW5hbnQiOiIkZGVmYXVsdCIsImxvYSI6MiwidGd0ZXhwIjoxNTg4MDgyMDQzfQ.-3GGokdehO9YlicEqB8CMAn6H5wfGH6ehiIguB0fMOc)

hsnfirooz committed:
added insurance full terms and conditions
